Assets/Editor Toolbox/Editor/SceneView/ToolboxEditorSceneViewObjectSelector.cs

```diff
@@ -0,0 +1,344 @@
+﻿using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+
+namespace Toolbox.Editor.SceneView
+{
+    public class ToolboxEditorSceneViewObjectSelector : EditorWindow
+    {
+        private List<GameObject> gameObjects;
+        private List<string> gameObjectPaths;
+
+        private const float sizeXPadding = 2f;
+        private const float sizeYPadding = 2f;
+        private const float buttonYSpacing = 0.0f;
+        private const float buttonYSize = 20f;
+        private const float sizeXOffset = -30f;
+
+        private GameObject highlightedObject;
+        private Vector2 size;
+        private Vector2 buttonSize;
+        private GUIStyle buttonStyle;
+        private Color selectionColor = new Color(0.50f, 0.70f, 1.00f);
+
+        private int shiftMinSelectionID = -1;
+        private int shiftMaxSelectionID = -1;
+        private int shiftLastID = -1;
+
+        private GameObject HighlightedObject
```

Comment:
I need to add Coding Conventions document :D Properties should be below all methods